virttest/utils_libvirt/libvirt_network.py

```diff
@@ -89,6 +112,17 @@ def check_established(params):
     else:
         cmdRes = remote.run_remote_cmd(cmd, params)
 
+    if check_network_conn_num:
```

Comment:
This variable name seems have two purpose. But it looks confused to me. ' check_xxx'  usually is for boolean with yes or no. So I suggest you change the name to 'expected_netwok_conn_num', for example.

Reply:
Updated to 'expected_netwok_conn_num', thanks.

Reply:
Could you help update again?  I did not notice the type, it should be `expected_network_conn_num`

Reply:
Updated, thanks.